src/Umbraco.Core/Persistence/Repositories/DataTypeDefinitionRepository.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;

```

Comment:
No, I did a `git fetch upstream` then merged it into `dev-v7`.

I'm sure it's due to my previous PR, but unsure of how to fix it from here.